dist/common/scripts/scylla_prepare

```diff
@@ -20,8 +20,6 @@
 # You should have received a copy of the GNU General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
```

Comment:
Are these two lines related to this patch?
I see a similar complain from @nyh below about a different hunk.
I join Nadav's recommendation. - Send cleanups in a separate patch, please.

Reply:
I'll remove this and the one Nadav pointed out from the PR.